findoutlie/metrics.py

```diff
@@ -29,4 +28,10 @@ def dvars(img):
     # You may be be able to solve this in four lines, without a loop.
     # But solve it any way you can.
     # This is a placeholder, replace it to write your solution.
-    raise NotImplementedError('Code up this function')
+    data = img.get_fdata()
+    prev_vol = data[..., 1:]
+    this_vol = data[..., 0:-1]
```

Comment:
(style) no need to write the zero-index here
```suggestion
    this_vol = data[..., :-1]
```